enterprise/server/workflow/service/service.go

```diff
@@ -45,6 +48,8 @@ const (
 )
 
 var (
+	fetchWorkflowConfigBeforeExecution = flag.Bool("fetch_workflow_config_before_execution", false, "Whether to fetch the workflow config before executing the workflow and spawn the CI runner based on the workflow config.")
```

Comment:
just checking -- do we need to flag this out and do you plan to remove the flag after this is released?

Reply:
Will remove this flag since we have enough time before the next release to test it, and it's just as easy to revert this behavior by pasting in a bit of code, as it is to switch the flag on / off if something goes wrong.